app/src/main/res/values/strings.xml

```diff
@@ -93,6 +93,7 @@
     <string name="permission_editing_not_allowed">Editing is not allowed</string>
     <string name="permission_deleting_not_allowed">Deleting is not allowed</string>
     <string name="permission_pinning_not_allowed">Pinning is not allowed</string>
+    <string name="permission_image_picking_not_allowed">Image cannot be accessed without read storage permissions</string>
```

Comment:
Could you add to the strings.xml in `values-pt-rBR` the following value:

`<string name="permission_image_picking_not_allowed">A imagem não pode ser acessada sem a permissão para ler o armazenamento.</string>`

Reply:
This is already added.